src/styles/media.scss

```
@@ -0,0 +1,17 @@
+@media screen and (max-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(3, minmax(200px, 1fr));
+  }
+}
+
+@media screen and (max-width: 768px) {
```

according to the task, there should be three columns starting at 768px. so you have to change value:
```suggestion
@media screen and (max-width: 767px) {
```

---

please check other places as well